data/movie/The Peanuts Movie (2015)/collectors-edition/disc02-summary.txt

```diff
@@ -0,0 +1,450 @@
+File name: Snoopy's Playlist - Skating
+
+Name: Snoopy's Playlist - Linus and Lucy
+Type: Extra
+Comment: G3
+Source title ID: 28
+Duration: 0:00:45
+Chapters count: 2
+Size: 37.1 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Linus and Lucy.mkv
+
+Name: Snoopy's Playlist - Better When I'm Dancin'
+Type: Extra
+Comment: G4
+Source title ID: 29
+Duration: 0:01:29
+Chapters count: 2
+Size: 71.1 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Better When I'm Dancin'.mkv
+
+Name: Snoopy's Playlist - Linus and Lucy (Reprise)
+Type: Extra
+Comment: G5
+Source title ID: 30
+Duration: 0:00:16
+Chapters count: 2
+Size: 13.8 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Linus and Lucy (Reprise).mkv
+
+Name: Snoopy's Playlist - Good To Be Alive
+Type: Extra
+Comment: G6
+Source title ID: 38
+Duration: 0:00:21
+Chapters count: 2
+Size: 17.6 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Good To Be Alive.mkv
+
+Name: Snoopy's Playlist - Better When I'm Dancin' (Reprise)
+Type: Extra
+Comment: H1
+Source title ID: 39
+Duration: 0:01:19
+Chapters count: 2
+Size: 64.9 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Better When I'm Dancin' (Reprise).mkv
+
+Name: Snoopy's Playlist - Wingwalking / Fifi's Theme
+Type: Extra
+Comment: H2
+Source title ID: 31
+Duration: 0:02:09
+Chapters count: 2
+Size: 105.2 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Wingwalking and Fifi's Theme.mkv
+
+Name: Snoopy's Playlist - That's What I Like
+Type: Extra
+Comment: H3
+Source title ID: 32
+Duration: 0:01:38
+Chapters count: 2
+Size: 77.5 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - That's What I Like.mkv
+
+Name: Snoopy's Playlist - Curse You Red Baron
+Type: Extra
+Comment: H4
+Source title ID: 33
+Duration: 0:03:22
+Chapters count: 2
+Size: 156.5 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Curse You Red Baron.mkv
+
+Name: Snoopy's Playlist - Christmas Time Is Here
+Type: Extra
+Comment: H5
+Source title ID: 34
+Duration: 0:00:12
+Chapters count: 2
+Size: 9.1 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Christmas Time Is Here.mkv
+
+Name: Snoopy's Playlist - Never Give Up / Fifi's Theme
+Type: Extra
+Comment: H6
+Source title ID: 35
+Duration: 0:03:48
+Chapters count: 2
+Size: 185.8 MB
+Segment count: 2
+Segment map: 1-2,3
+File name: Snoopy's Playlist - Never Give Up and Fifi's Theme.mkv
+
+Name: Snoopy's Playlist - Linus and Lucy (Carnival)
+Type: Extra
+Comment: I1
+Source title ID: 36
+Duration: 0:01:00
+Chapters count: 2
+Size: 46.1 MB
+Segment count: 2
+Segment map: 1,2
+File name: Snoopy's Playlist - Linus and Lucy (Carnival).mkv
+
+Name: Snoopy's Playlist - End Credits
+Type: Extra
+Chapters:
+- Better When I'm Dancin'
+- Good To Be Alive
+- Snow Day
+- Skating
+Comment: I2
+Source title ID: 37
+Angle: 1
+Duration: 0:10:30
+Chapters count: 2
```

Comment:
Something seems off here - you specify 4 chapter names but the Chapters count is only 2. Any idea?